commit-graph.c

```diff
@@ -798,6 +799,7 @@ struct write_commit_graph_context {
 		 bloom:1;
 
 	const struct split_commit_graph_opts *split_opts;
+	uint32_t total_bloom_filter_size;
```

Comment:
So `filter->len` is the number of ui64 entries in `filter->data` and thus `ctx->total_bloom_filter_size` is size in bytes.  (not complaining, just observing and looking for math errors or pinch-points)